src/components/heading.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+class heading extends Component {
+  constructor() {
+    super();
+    this.state = { add: "" };
+    // this.event = this.event.bind(this);
```

Comment:
I'd suggest removing unused comments for cleaner code.

Reply:
sure will do that in future, reason leaving that so, that helps me know what other things I have tried.